report/src/problem_analysis/hci/prototyping.tex

```diff
@@ -0,0 +1,43 @@
+\section{Prototyping} %specifict prototyping
+A prototype is an instantiation of a design concept meant for describing, refining, testing, and depicting the design.\cite{BUXTON2007139_prototyping}
+
+When designing prototypes, the emphasis is not on the discovery of new knowledge and user needs, but to examine whether or not the proposed design constitutes a suitable hypothesis for the design of the product\cite{nielsen-norman-prototype-low-vs-high,BUXTON2007139_prototyping}
+That is, the design must allow for pleasurable interaction and suit user needs to a degree deemed adequate by both user and designer.
+
+In general, two categories of prototypes exist: High Fidelity (hifi) and Low Fidelity (lofi).
+Lofi prototypes are typically limited in their function and are used to emphasize concepts within the design, alternative design solutions, and model the general interaction with the artifact or system\cite{low-vs-high-fidelity-prototype}.
+These prototypes are helpful when performing ideation\cite{nielsen-norman-ideation} or when establishing user capabilities in terms of interaction with the system \cite{usefullness-of-different-prototypes,low-vs-high-fidelity-prototype}.
+Lofi prototypes should be used early in the design process to establish the overall direction for a product design. 
+Therefore, the creation of these prototypes and the following evaluations using them should not be a time-consuming or expensive process. \cite{usefullness-of-different-prototypes,low-vs-high-fidelity-prototype} 
+Lofi prototypes are not ideal when evaluating user interaction as they are used to present a design idea to the user, and thus are not interacted with by the user\cite{low-vs-high-fidelity-prototype}.
+
+Unlike lofi prototypes, hifi prototypes provide near-complete functionality and interactivity with a design closely representing the final product.
+They enable users to interact with buttons, icons, and input fields.
+However, due to their detailed and interactive nature, constructing hifi prototypes is more time-consuming than creating lofi prototypes. 
+This makes hifi prototypes more suitable for examining users' interactions with the designed artifact, than exploring new design ideas. \cite{nielsen-norman-prototype-low-vs-high,low-vs-high-fidelity-prototype}
+
+
+
+Since the stakeholder has already suggested ideas for a user interface design based on existing systems, we already have a direction for an initial product design. 
+As we want to find a common denominator for the stakeholders' problems described in Section \ref{sec:understanding_the_problem} we need to understand how users interact with a product design resembling a potential solution.
+To this end, a hifi prototype is more suitable than a lofi prototype \cite{low-vs-high-fidelity-prototype}.
```

Comment:
```suggestion
To understand how users interact with a potential solution design, creating a high-fidelity prototype would be more appropriate than a low-fidelity prototype\cite{low-vs-high-fidelity-prototype}.
```
I tried simplifying this sentence, but I still feel like you need to argue why we decided to chose to do a Hi-Fi prototype. A cite is not an argument, and we should explain our reasoning (time constraints, modern tooling, etc.)